src/components/DatePicker/DatePicker.tsx

```diff
@@ -61,10 +62,26 @@ export interface DatePickerProps extends DatePickerWrapperProps {
   dateType?: DatePickerTypes;
   /** The date format to display. Defaults to "yyyy-MM-dd". */
   dateFormat?: string;
-  /** Populates the `HelperErrorText` component in the "From" `TextInput` component. */
+  /** Populates the `HelperErrorText` component in this component. */
   helperText?: string;
+  /** Populates the `HelperErrorText` component in the "From" `TextInput` component. */
+  helperTextFrom?: string;
   /** Populates the `HelperErrorText` component in the "To" `TextInput` component. */
   helperTextTo?: string;
+  /** The initial date value. This must be in the mm/dd/yyyy format. */
+  initialDate?: string;
+  /** The initialTo date value used for date ranges.
+   * This must be in the mm/dd/yyyy format.
+   */
+  initialDateTo?: string;
+  /** The minimum date value that applies to both input fields.
+   * This must be in the mm/dd/yyyy format.
```

Comment:
@EdwinGuzman Do you have any concerns that there could be confusion around formatting this string and using the `dateFormat` prop?

Reply:
Sort of. The visual representation is the `dateFormat` which is displayed in the input field, but any js-related date information needs to be in the specific format so that the js date parser can work properly. It should still be possible to say "I only want the year to display" but have to pass in an initial date of "1/1/2021". It is a bit annoying but the workaround is more annoying to build in.